sidekiq.gemspec

```diff
@@ -25,4 +25,5 @@ Gem::Specification.new do |gem|
   gem.add_dependency "redis", ">= 4.2.0"
   gem.add_dependency "connection_pool", ">= 2.2.2"
   gem.add_dependency "rack", "~> 2.0"
+  gem.add_dependency "after_commit_everywhere"
```

Comment:
I think that it should be in `Gemfile`

Reply:
Ah, right, if we want it to be completely optional.